util/Migrator/DbScripts_transition/2023-12-06_02_ManagersEditAssignedCollectionUsers.sql

```diff
@@ -0,0 +1,25 @@
+-- Update [dbo].[CollectionUser] with [Manage] = 1 for all users with Manager role or 'EditAssignedCollections' permission
+UPDATE cu
+SET cu.[ReadOnly] = 0,
+    cu.[HidePasswords] = 0,
+    cu.[Manage] = 1
+FROM [dbo].[CollectionUser] cu
+INNER JOIN [dbo].[OrganizationUser] ou
+    ON cu.[OrganizationUserId] = ou.[Id]
+WHERE (ou.[Type] = 3 OR (ou.[Permissions] IS NOT NULL AND
+    ISJSON(ou.[Permissions]) > 0 AND JSON_VALUE(ou.[Permissions], '$.editAssignedCollections') = 'true'))
+
+-- Insert rows to [dbo].[CollectionUser] for Managers and users with 'EditAssignedCollections' permission assigned to groups with collection access
```

Comment:
What if they also have direct access (i.e. a CollectionUser entry)? This will create duplicate rows.